pkg/ocm/ocm.go

```diff
@@ -18,6 +19,9 @@ const (
 	stagingURL       = "https://api.stage.openshift.com"
 	integrationURL   = "https://api.integration.openshift.com"
 	productionGovURL = "https://api.openshiftusgov.com"
+
+	ocmConfigDest      = "/root/.config/ocm/ocm.json"
+	ocmConfigMountOpts = "rw"
```

Comment:
RW allows for tokens to be reused until they expire, so you don't have to login every single time